### PR TITLE
FORMS-7736 Wizard and Tabs on top becomes invisible in authoring when…

### PR DESCRIPTION
**Pull request by sakshi-arora1**
… hide component is selected.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by nit23uec** [comment]
I'd prefer to handle it in the sling model where we detect authoring mode and return visible property to be always true.
Any reason why we can't do the same for wizard/tabs also?

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/page/v1/page/customheaderlibs.html [hunk 1]
```diff
@@ -20,4 +20,5 @@
 </sly>
 <sly data-sly-use.clientlib="core/wcm/components/commons/v1/templates/clientlib.html">
     <sly data-sly-test="!${wcmmode.edit}" data-sly-call="${clientlib.css @ categories='core.forms.components.runtime.all'}"/>
+    <sly data-sly-test="${wcmmode.edit}" data-sly-call="${clientlib.css @ categories='core.forms.components.authoring'}"/>
```
[the remark this answers is not in the document]
Comment (sakshi-arora1) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/page/v1/page/customheaderlibs.html:23]: As we are including runtime clientlib, in a similar fashion authouring clientlib will need to be added (This becomes pre-requisite if sites page is to use forms component), will need documentation.

[the remark this answers is not in the document]
Comment (sakshi-arora1) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/page/v1/page/customheaderlibs.html:23]: cc: @ujjwal5

**Comment by sakshi-arora1** [comment]
@nit23uec , @rismehta The other thing which we can do is add this authouring library to form container component . If we agree on this I will make the further relevant changes.

**Comment by sakshi-arora1** [comment]
> I'd prefer to handle it in the sling model where we detect authoring mode and return visible property to be always true.
> Any reason why we can't do the same for wizard/tabs also?

@nit23uec We are adding runtime for tabs to show active tab and navigate in between tabs. So this irrespective of fact handled in sightly, runtime will take precedence.

**Comment by rismehta** [comment]
> @nit23uec , @rismehta The other thing which we can do is add this authouring library to form container component . If we agree on this I will make the further relevant changes.

We should add it at component level in authoring, so that changes at other places are minimal. Authoring specific client libraries should be avoided

**Pull request closed without merging**